wsmaster/codenvy-hosted-api-license/src/test/java/com/codenvy/api/license/CodenvyLicenseTest.java

```diff
@@ -14,36 +14,50 @@
  */
 package com.codenvy.api.license;
 
-import com.codenvy.api.license.CodenvyLicense;
-import com.codenvy.api.license.LicenseFeature;
 import com.google.common.collect.ImmutableMap;
+import com.license4j.License;
+
+import org.mockito.Mock;
+import org.mockito.testng.MockitoTestNGListener;
+import org.testng.annotations.BeforeMethod;
 import org.testng.annotations.DataProvider;
+import org.testng.annotations.Listeners;
 import org.testng.annotations.Test;
 
 import java.util.Map;
 
+import static org.mockito.Mockito.when;
 import static org.testng.Assert.assertEquals;
 
 /**
  * @author Anatoliy Bazko
  * @author Dmytro Nochevnov
  * @author Alexander Andrienko
  */
+@Listeners(value = {MockitoTestNGListener.class})
 public class CodenvyLicenseTest {
 
+    @Mock
+    private License license;
```

Comment:
It's better to name it **license4j** to avoid confusion with **codenvyLicense** variable.